src/test/end-to-end/paths/crossBrowserTests.js

```diff
@@ -0,0 +1,23 @@
+const testConfig = require('./../../config');
+const {navigateCase} = require("../helpers/caseHelper");
+const {eventNames} = require('../pages/common/constants.js');
+const {acceptCaseTest, caseDetails} = require("../helpers/caseHelper");
+
+Feature('Leeds Singles Case and move to Accepted state');
+// BeforeSuite(async ({I}) => caseId = await createCaseInCcd('test/end-to-end/data/ccd-case-basic-data.json'));
```

Comment:
Should this be disabled? this looks to be creating the case for the following tests

Reply:
no... but fixed the issue